lib/simulator-xcode-9.js

```diff
@@ -272,31 +281,40 @@ class SimulatorXcode9 extends SimulatorXcode8 {
   }
 
   /**
-   * Get the current state of Touch ID Enrollment feature.
+   * Get the current state of a biometric (touch id, face id) enrollment.
    * @override
    */
-  async isTouchIDEnrolled () {
+  async isBiometricEnrolled (biometricName='touchId') {
```

Comment:
to make this not a breaking change you could simply keep the `isTouchIDEnrolled` method and `enrollTouchID` method and have them call the more general one. not sure if we care.

Reply:
I thought that too, but since appium-xcuitest-driver is the only dependent, I think bumping the major version should be okay.